diff --git a/library/stats.py b/library/stats.py
index 9af8506..43655df 100644
--- a/library/stats.py
+++ b/library/stats.py
@@ -818,16 +818,16 @@ class Custom:
                 if string_value is None:
                     string_value = str(numeric_value)
 
-                # Display text
-                theme_data = config.THEME_DATA['STATS']['CUSTOM'][custom_stat].get("TEXT", None)
-                if theme_data is not None and string_value is not None:
-                    display_themed_value(theme_data=theme_data, value=string_value)
-
                 # Display graph from numeric value
                 theme_data = config.THEME_DATA['STATS']['CUSTOM'][custom_stat].get("GRAPH", None)
                 if theme_data is not None and numeric_value is not None and not math.isnan(numeric_value):
                     display_themed_progress_bar(theme_data=theme_data, value=numeric_value)
 
+                # Display text
+                theme_data = config.THEME_DATA['STATS']['CUSTOM'][custom_stat].get("TEXT", None)
+                if theme_data is not None and string_value is not None:
+                    display_themed_value(theme_data=theme_data, value=string_value)
+
                 # Display radial from numeric and text value
                 theme_data = config.THEME_DATA['STATS']['CUSTOM'][custom_stat].get("RADIAL", None)
                 if theme_data is not None and numeric_value is not None and not math.isnan(
